test/e2e/benchmark/throughput.go

```diff
@@ -138,54 +143,23 @@ func runBenchmarkTest(logger *log.Logger, testName string, manifest Manifest) er
 }
 
 func TwoNodeBigBlock8MB(logger *log.Logger) error {
-	manifest := bigBlockManifest
-	manifest.MaxBlockBytes = 8 * testnet.MB
+	manifest, err := getBigBlockManifest()
+	testnet.NoError("failed to get manifest", err)
 	return runBenchmarkTest(logger, "TwoNodeBigBlock8MB", manifest)
 }
 
 func TwoNodeBigBlock8MBLatency(logger *log.Logger) error {
-	manifest := bigBlockManifest
-	manifest.MaxBlockBytes = 8 * testnet.MB
+	manifest, err := getBigBlockManifest()
+	testnet.NoError("failed to get manifest", err)
 	manifest.EnableLatency = true
-	manifest.LatencyParams = LatencyParams{70, 0}
 	return runBenchmarkTest(logger, "TwoNodeBigBlock8MBLatency", manifest)
 }
 
-func TwoNodeBigBlock32MB(logger *log.Logger) error {
-	manifest := bigBlockManifest
-	manifest.MaxBlockBytes = 32 * testnet.MB
-	return runBenchmarkTest(logger, "TwoNodeBigBlock32MB", manifest)
-}
-
-func TwoNodeBigBlock64MB(logger *log.Logger) error {
-	manifest := bigBlockManifest
-	manifest.MaxBlockBytes = 64 * testnet.MB
-	return runBenchmarkTest(logger, "TwoNodeBigBlock64MB", manifest)
-}
-
 func LargeNetworkBigBlock8MB(logger *log.Logger) error {
-	manifest := bigBlockManifest
-	manifest.MaxBlockBytes = 8 * testnet.MB
+	manifest, err := getBigBlockManifest()
+	testnet.NoError("failed to get manifest", err)
 	manifest.Validators = 50
 	manifest.TxClients = 50
 	manifest.BlobSequences = 2
 	return runBenchmarkTest(logger, "LargeNetworkBigBlock8MB", manifest)
 }
-
-func LargeNetworkBigBlock32MB(logger *log.Logger) error {
```

Comment:
I don't think this was part of the exported Go API https://pkg.go.dev/github.com/celestiaorg/celestia-app/v2@v2.2.0/test/e2e/benchmark so we can probably drop the `!` from the PR title